Lib/test/test_pty.py

```diff
@@ -75,6 +76,14 @@ def _readline(fd):
 
 def expectedFailureIfStdinIsTTY(fun):
     # avoid isatty() for now
+    PLATFORM = platform.system()
```

Comment:
I don't understand the rationale for skipping the test only on Gentoo. Would you mind to write a comment to explaining the rationale? Why not skipping the test on Ubuntu? Is it a temporary workaround until someone can investigate the root issue? Please add a reference to bpo-41818 in the command.

Reply:
No, it is the reverse: Gentoo passes the test, all other tested Linux distributions fail.
That's why `expectedFailure()` is applied for each platform where stdin is a TTY except Gentoo.

I'll add a comment.